### DISCUSSION TITLE
Storybook 8

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco.UI#854] [Ideas]
I noticed Storybook 8 is avaiable: https://storybook.js.org/blog/storybook-8/ which seems to be faster, enhanced on mobile etc.

![image](https://github.com/user-attachments/assets/d2a6d30a-4dcb-4790-b44f-dc008a7f8b2c)

Currently we use Storybook v7.6.14 (or latest minor).

@iOvergaard would it be possible to upgrade/migrate?

**Comment by iOvergaard** [comment]
It's something we intend to look into at some point for sure. We need to update both repositories, but I think the biggest hurdle is to update all our stories to `StoryObj` first. I am not entirely sure if that is a requirement of Storybook 8, but the old format has been deprecated since V7 so I would like to get that done first.

**Reply by iOvergaard** [reply]
If anyone wants to help convert from `Story` to `StoryObj`, that is definitely up for grabs!

I think the stories for Color Picker is one of the better examples located in `packages/uui-color-picker/lib/uui-color-picker.story.ts`
